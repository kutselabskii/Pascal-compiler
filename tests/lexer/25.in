'h'#123'world}'