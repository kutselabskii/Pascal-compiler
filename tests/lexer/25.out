Line Col  Type           Value                         Original                      
1    16   String         h{world}                      h{world}                      
1    16   End of file    EOF                                                         
